pkg/acctest/test.go

```diff
@@ -11,13 +11,14 @@ import (
 
 	"github.com/go-resty/resty/v2"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	terraform2 "github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
```

Comment:
I don't get why you have the same import twice, one aliased

Reply:
It's not the same import. 
We need an alias because of this conversion:
```
configErr := Provider.Configure(context.Background(), (*terraform2.ResourceConfig)(terraform.NewResourceConfigRaw(nil)))
```